release_images.yml

```diff
@@ -4,22 +4,38 @@ release_images:
     framework: "tensorflow"
     version: "2.2.0"
     training:
-      device_types: ["cpu", "gpu"]
```

Comment:
I thought we were going to re-release inference cu102 as well?

Reply:
TF22 cu102 Inference images have already been released together with TF22 cu101 Training images yesterday.

Reply:
Added them back